Software/real_time/ROS_RoboBuggy/src/robobuggy/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ link_directories(/usr/local/lib)
 ## Uncomment this if the package has a setup.py. This macro ensures
 ## modules and global scripts declared therein get installed
 ## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
-# catkin_python_setup()
```

Comment:
remove instead of commenting

Reply:
Should we remove this? Line 23 says "Uncomment this", so should we keep it or remove it?

Reply:
Whoops, didn't see that, good catch